packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -32,12 +34,16 @@ export class ConnectorController {
      * @ignore
      */
     #editorAPI: EditorAPI;
+    config: ConfigType;
+    connectorCompatibilityTools: ConnectorCompatibilityTools;

```

Comment:
To make property private
```suggestion
    #connectorCompatibilityTools: ConnectorCompatibilityTools;

```